web/src/components/data-fields/Industry.test.tsx

```diff
@@ -90,7 +90,10 @@ describe("<Industry />", () => {
     });
 
     const businessPersonas = ["STARTING", "FOREIGN"];
-    EssentialQuestions.map((el) => {
+    const nonConditionalEssentialQuestions = EssentialQuestions.filter((eq) => {
+      return eq.fieldName !== "residentialConstructionType";
+    });
+    nonConditionalEssentialQuestions.map((el) => {
```

Comment:
These tests were failing because they were searching for a conditional essential question. Rather than update the majority of test cases, I pull out the conditional essential questions to test switching industries separately.